src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,51 @@
+package calculator
+
+import calculator.exception.NegativeNumericException
+import calculator.exception.NotNumericException
+
+class StringAddCalculator {
+
```

Comment:
객체 내부에 상태값을 갖지 않아 싱글턴 객체인 object로 만들어도 괜찮다는 생각이 드네요 :)
```suggestion
object StringAddCalculator {

```